prow/plugins/golint/golint_test.go

```diff
@@ -224,27 +224,48 @@ func TestLintCodeSuggestion(t *testing.T) {
 		pullFiles  map[string][]byte
 		comment    string
 	}{
-		{
+		{ // Check names with underscore
```

Comment:
Instead of using comments, please add a field to the test case struct so that we can expose the testcase name in the error messages (otherwise the errors could be ambiguous). The test case name can be logged like this: https://github.com/kubernetes/test-infra/blob/27399df3e76d40931c495fee7693b88d23d95a15/prow/plugins/require-matching-label/require-matching-label_test.go#L246
or like this: https://github.com/kubernetes/test-infra/blob/52799d18012fa222080d1664e50b5581e120ba1f/prow/config/branch_protection_test.go#L69